<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmIgra</class>
 <widget class="QWidget" name="FrmIgra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mala matematika</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frameTrazeniBroj">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblTrazeniBroj1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTrazeniBroj2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>57</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTrazeniBroj3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>57</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>NOVA IGRA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMinus">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBroj6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPuta">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPodeljeno">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZagradaOtvorena">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZagradaZatvorena">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnObrisi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Obrisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPotvrdi">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Potvrdi</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFormulaNatpis">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Formula:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameFormula1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameRezultat1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblRezultatNatpis">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Rezultat:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
